### PR TITLE
Np 9329 dlr auth customer handling

### PR DESCRIPTION
**Pull request by truhacevkir**

### CODE REVIEW

File: customer-commons/src/main/java/no/unit/nva/customer/model/ApplicationDomain.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.net.URI;
+import java.util.Arrays;
+import nva.commons.core.SingletonCollector;
+
+public enum ApplicationDomain {
+    
+    NVA(URI.create("https://nva.org"));
```
Comment (brinxmat) [inline customer-commons/src/main/java/no/unit/nva/customer/model/ApplicationDomain.java:11]: We should change this -> but I don't know to what the final domain will be…at least nva.unit.no for now

File: user-access-service/build.gradle [hunk 2]
```diff
@@ -24,6 +24,7 @@ dependencies {
     implementation libs.slf4j
     implementation libs.bundles.aws.lambda
 
+    testImplementation group: 'io.github.ganadist.sqlite4java', name: 'libsqlite4java-osx-aarch64', version: '1.0.392'
```
Comment (brinxmat) [inline user-access-service/build.gradle:27]: Can be removed

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: user-access-service/build.gradle (hunk 2 above)
Comment (truhacevkir) [reply user-access-service/build.gradle:27]: done

File: customer-commons/src/main/java/no/unit/nva/customer/model/ApplicationDomain.java (hunk 1 above)
Comment (truhacevkir) [reply customer-commons/src/main/java/no/unit/nva/customer/model/ApplicationDomain.java:11]: done

**Pull request closed without merging**